config/initializers/active_record_query_trace.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+ActiveRecordQueryTrace.enabled = true if Rails.env.development? || Rails.env.test?
```

Comment:
[pending deletion if we use Rails config for this instead, original question above]